generator/src/main/resources/line-bot-sdk-nodejs-generator/apiBody/normal.pebble

```diff
@@ -10,9 +10,15 @@
     {% if op.hasBodyParam %}const params = {{op.bodyParam.paramName}};
     {% elseif op.hasFormParams %}const formParams = {
         {% for param in op.formParams -%}
-            "{{param.paramName}}": {{param.paramName}},
+            "{{param.baseName}}": {{param.paramName}},
         {% endfor %}
         };
+        // Remove properties with undefined or empty values
```

Comment:
Why was this change included in this commit?

Reply:
Let me remove this~